CodePush.m

```diff
@@ -177,6 +179,18 @@ - (void)startRollbackTimer:(int)rollbackTimeout
     resolve(@(isFailedHash));
 }
 
+RCT_EXPORT_METHOD(isFirstRun:(NSString *)packageHash
+                  resolve:(RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject)
+{
+    NSError *error;
+    BOOL isFirstRun = (nil != packageHash
+                  && [packageHash length] > 0
+                  && [packageHash isEqualToString:[CodePushPackage getCurrentPackageHash:&error]]
```

Comment:
Based on the developer guide, I think this is how methods that can return errors are handled?

https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ErrorHandlingCocoa/CreateCustomizeNSError/CreateCustomizeNSError.html#//apple_ref/doc/uid/TP40001806-CH204-BAJIIGCC